### ISSUE TITLE
Deprecation warning

### ISSUE THREAD

**Issue by brunowego** [nucleartide/ember-outside-click#4]
FYI

```sh
DEPRECATION: Ember.K is deprecated in favor of defining a function inline. [deprecation id: ember-metal.ember-k] See http://emberjs.com/deprecations/v2.x#toc_code-ember-k-code for more details.
```

**Comment by nucleartide** [comment]
Will fix, thanks for the heads up!

**Comment by nucleartide** [comment]
@brunowego Released v1.1.1 to npm. Lemme know if there are any other problems. 👍
